eng/native/configureplatform.cmake

```diff
@@ -214,6 +214,12 @@ if(CLR_CMAKE_HOST_OS STREQUAL sunos)
     endif(SUNOS_KERNEL_KIND STREQUAL illumos OR CMAKE_CROSSCOMPILING)
 endif(CLR_CMAKE_HOST_OS STREQUAL sunos)
 
+if(CLR_CMAKE_HOST_OS STREQUAL haiku)
+    set(CLR_CMAKE_HOST_UNIX 1)
+    set(CLR_CMAKE_HOST_UNIX_AMD64 1)
```

Comment:
Does Haiku support building on x64 host only?

Reply:
For the purpose of this PR and the ones in the near future, yes.